modules/users/server/config/strategies/facebook.js

```diff
@@ -30,7 +30,7 @@ module.exports = function(config) {
 				displayName: profile.displayName,
 				email: profile.emails[0].value,
 				username: profile.username,
-				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value : undefined,
+				profileImageURL: (profile.id) ? 'https:graph.facebook.com/' + profile.id + '/picture?type=large' : undefined,
```

Comment:
is it actually https:graph or is that a typo and should read really https://graph...?